packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1167,6 +1194,13 @@ class DropdownButton<T> extends StatefulWidget {
   ///    relative to text direction.
   final AlignmentGeometry alignment;
 
+  /// Applies a borderRadius to the menu.
+  /// The first menu item uses topLeft and topRight of the borderRadius.
```

Comment:
The radii of the first menu item's top left and right corners are defined by the corresponding properties of the [borderRadius].  Similarly, the radii of the last menu item's bottom and right corners are defined by the corresponding properties of the [borderRadius].